examples/backends/vllm/launch/xpu/agg_tracing_xpu.sh

```
@@ -0,0 +1,68 @@
+
+# run ingress
+# dynamo.frontend accepts either --http-port flag or DYN_HTTP_PORT env var (defaults to 8000)
+export OTEL_SERVICE_NAME=dynamo-frontend
+python -m dynamo.frontend &
+
+# run worker
+# --enforce-eager is added for quick deployment. for production use, need to remove this flag
+export OTEL_SERVICE_NAME=dynamo-worker-vllm
+DYN_SYSTEM_PORT=${DYN_SYSTEM_PORT:-8081} \
+    python -m dynamo.vllm --model "$MODEL" --enforce-eager \
+    # --block-size $BLOCK_SIZE \
```

Shellcheck (0.11.0)

[error] This backslash is part of a comment and does not continue the line.

(SC1143)